app/src/main/java/nu/yona/app/utils/AppUtils.java

```diff
@@ -79,6 +82,7 @@
 	private static ScheduledExecutorService scheduler;
 	private static final YonaReceiver receiver = new YonaReceiver();
 	private static int trialCertificateCount = 0, trialVPNCount = 0;
+	private static final Handler mHandler = new Handler();
```

Comment:
What's this ``m`` prefix for? Please give the variable a speaking name.

Reply:
Done.